src/core_plugins/kibana/public/dashboard/panel/panel.js

```diff
@@ -117,6 +117,10 @@ uiModules
           $scope.$watchCollection('panel.sort', function () {
             $scope.saveState();
           });
+
+          $scope.setSortOrder = function setSortOrder(columnName, direction) {
```

Comment:
I wonder if we can get rid of the watch above if we move `$scope.saveState()` in here.  As long as the state is updated from the initial setting above.  Do you think getting rid of the watch would increase readability?